ocs_ci/utility/utils.py

```
@@ -1051,27 +1051,73 @@ def get_url_content(url, **kwargs):
+            log.warning(f"Nightly release service unavailable for {version}")
+
+            # Extract major.minor version (e.g., "4.22.0-0.nightly" -> "4.22")
+            try:
+                major_minor = version.split(".nightly")[0]  # "4.22.0-0"
+                major_minor = ".".join(major_minor.split(".")[0:2])  # "4.22"
+
+                # Check if this major.minor version has GA releases available
+                if is_ocp_version_gaed(major_minor):
+                    log.warning(
+                        f"OpenShift CI nightly service is unavailable. "
+                        f"Falling back to latest GA version of {major_minor} instead of nightly."
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)